src/Auth0Client.ts

```diff
@@ -209,12 +209,19 @@ export class Auth0Client {
   private readonly isAuthenticatedCookieName: string;
   private readonly nowProvider: () => number | Promise<number>;
   private readonly httpTimeoutMs: number;
-  private readonly useFormData: boolean;
+  private readonly options: Auth0ClientOptions;
 
   cacheLocation: CacheLocation;
   private worker: Worker;
 
-  constructor(private options: Auth0ClientOptions) {
+  private readonly defaultOptions: Partial<Auth0ClientOptions> = {
+    useRefreshTokensFallback: false,
+    useFormData: true
+  }
+
+  constructor(options: Auth0ClientOptions) {
+    this.options = {...this.defaultOptions, ...options };
```

Comment:
```suggestion
    this.options = { ...this.defaultOptions, ...options };
```

Reply:
@ewanharris if you install the husky precommit hook, it should fix this stuff

Reply:
Gah my bad! Used to husky installing automatically, thanks!